db/migrate/20160414185708_create_container_label_tag_mappings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateContainerLabelTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_label_tag_mappings do |t|
+      t.string :labeled_resource_type
```

Comment:
I called it `labeled_` for consistency with @simon3z's suggestion of `label_name` and `label_value`; I'm happy to adopt whatever naming people tell me :-)